python/surf/devices/transceivers/_Qsfp.py

```diff
@@ -124,6 +124,37 @@ def __init__(self,**kwargs):
                 dependencies = [self.TxPwrRaw[2*i+0],self.TxPwrRaw[2*i+1]],
             ))
 
+        for i in range(4):
+            self.add(pr.RemoteVariable(
+                name         = f'TxDisable[{i}]',
+                description  = 'Tx_Disable bit that allows software disable of transmitters, Writing 1 disables the laser of the channel',
+                offset       = (86 << 2),
+                bitSize      = 1,
+                bitOffset    = i,
+                mode         = 'RW',
+                base         = pr.Bool,
+            ))
+
+        self.add(pr.RemoteVariable(
+            name         = 'PowerOverride',
+            description  = '1: allows setting power mode with hardware, 0: allows setting power mode with software',
+            offset       = (93 << 2),
+            bitSize      = 1,
+            bitOffset    = 0,
+            mode         = 'RW',
+            base         = pr.Bool,
+        ))
+
+        self.add(pr.RemoteVariable(
+            name         = 'PowerMode',
+            description  = 'Power set to low power mode: 1 sets to LP mode if PowerOverride is 0',
```

Comment:
I think there is a typo: should be
`'Power set to low power mode: 1 sets to LP mode if PowerOverride is 1',`
check table 6-11 of the datasheet to confirm